install_config/install/advanced_install.adoc

```diff
@@ -512,6 +514,108 @@ xref:../../admin_guide/diagnostics_tool.adoc#additional-cluster-health-checks[Ad
+- link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html-single/managing_containers/#using_skopeo_to_work_with_container_registries[skopeo] for searching.
+
+System containers are therefore stored and run outside of the traditional
+`docker` service. For more details on system container technology, see
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers[Running System Containers] in the _Red Hat Enterprise Linux Atomic Host: Managing Containers_ documentation.
+
+You can configure your {product-title} installation to run certain components as
+system containers instead of their RPM or standard containerized methods.
+Currently, the `docker` and etcd components can be run as system containers in
+{product-title}.
+
+[[advanced-install-docker-system-container]]
+==== Running Docker as a System Container
+
+[IMPORTANT]
+====
+All system container components are Technology Preview features in
+{product-title} 3.6. They must not be used in production and they are not
+supported for upgrades to {product-title} 3.6. During this Technology Preview
+phase, they are only meant for use with new cluster installations in
+non-production environments.
+====
+
+The traditional method for using `docker` in an {product-title} cluster is an
+xref:../../install_config/install/host_preparation.adoc#installing-docker[RPM package installation]. For Red Hat Enterprise Linux (RHEL) systems, it must be
+specifically installed; for RHEL Atomic Host systems, it is provided by default.
+
+However, you can configure your {product-title} installation to alternatively
+run `docker` on node hosts as a system container. When using the system
+container method, the `container-engine` systemd service is used on the host
+instead of the `docker` service.
+
+To run `docker` as a system container, set the following cluster variable in
+your inventory file in the `[OSEv3:vars]` section:
+
+----
+openshift_docker_use_system_container=True
+----
+
+When using the system container method, the following inventory variables for
+`docker` are ignored:
+
+- `docker_version`
+- `docker_upgrade`
+
+Further, the following inventory variable must not be used:
+
+- `openshift_docker_options`
+
+You can also force `docker` in the system container to use a specific container
+registry instead of the default `registry.access.redhat.com`. This is similar
+to the options in
+xref:advanced-install-configuring-registry-location[Configuring a Registry
+Location] when using the traditional RPM-based `docker` installation.
+
+To override the default registry location, set the following cluster variable in
+your inventory file in the `[OSEv3:vars]` section:
+
+----
+openshift_docker_systemcontainer_image_registry_override="registry.example.com"
```

Comment:
Currently docker system container image `container-engine` is located in `rhel7` namespace instead of `openshift3`.

 And actually `openshift_docker_systemcontainer_image_registry_override` is a image prefix. So better to change the example to:

```
# For enterprise deployments, use the image prefix "registry.access.redhat.com/rhel7"
openshift_docker_systemcontainer_image_registry_override=registry.example.com/myrepo
```